web/alerting/status_gluster_afr.rst

```diff
@@ -21,8 +23,107 @@ Based on:
 Setup
 =====
 
-#. You need a gluster volume.
-   TODO: list of volumes from qe configurations which uses AFR translator
+#. You need a gluster arbiter volume.
+   (distributed, distributed-replicate) volumes uses AFR translator
+   to this categories falls also arbiter - distributed-replicate volume
+
+Test Steps
+==========
+
+.. test_action::
+    :step:
+        List *replica sets* - *subvolumes* of present arbiter volume
```

Comment:
Could you link a document (from gluster upstream project), which makes you think that *subvolume* is *a replica set* in this context? The link should be added into list of references (titled **Based On**) in Description section,

Reply:
That's not really from documentation. First @mkudlej points me to the output of *gluster get-state* command, where is stated that there's 6 subvolumes in my arbiter volume, which corresponds to number of replica sets. And when I used that knowledge it really works as expected, so I am pretty sure it's correct.

Reply:
BTW Only definition of subvolume I found in https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.2/html-single/administration_guide/ but this definition is not usable:
 Translator
    A translator connects to one or more subvolumes, does something with them, and offers a subvolume connection. 
Subvolume
    A brick after being processed by at least one translator.